### PR TITLE
Chore/initial setup cs

### PR DESCRIPTION
**Pull request by corysanders3**
Summary
- Created 5 new components (Nav, Home, Main, Teas, Card)
- Created a folder for each component, and all those folders live in a 'components' folder
- Setup initial imports and exports between the components
- setup BrowserRouter in the index.tsx

### CODE REVIEW

File: src/components/main/Main.tsx
Comment (tednaphil) [inline src/components/main/Main.tsx:1]: Refer to Asher's note about omitting this in favor of just using the `Home` component and utilizing the `main` jsx tag

**Review by tednaphil — APPROVED** [review]

File: src/components/main/Main.tsx
Comment (AsherSpurr) [inline src/components/main/Main.tsx:1]: We can delete -- the Main component -- this will be the Home component

**Review by EthanDuvall — APPROVED** [review]
Nicely done looks great !

File: src/components/home/Home.tsx [hunk 1]
```diff
@@ -1,5 +1,4 @@
 import './Home.css';
-import Main from '../main/Main';
```
Comment (tednaphil) [inline src/components/home/Home.tsx]: 😎

**Review by AsherSpurr — APPROVED** [review]
Rock on thx for deleting Main component

**Pull request merged**